examples/cephfs/storageclass.yaml

```diff
@@ -15,7 +15,7 @@ parameters:
   clusterID: <cluster-id>
 
   # CephFS filesystem name into which the volume shall be created
-  fsName: myfs
+  fsName: <cephfs-name>
```

Comment:
if you want to make this field as required one what about the secrets? in the documentation, we talk about the configurable parameters (i think it should be enough) this is just an example

Reply:
Madhu, what I mean was that the fs or the pool is an external entity to the CSI, which is coming from the ceph cluster.
For example, if you look just a few lines above we have `<cluster-id>` field and we have that as mandatory because we expect it changing right?

The Secret looks good to me in that context:
```
---                                                                     
apiVersion: v1                                                          
kind: Secret                                                            
metadata:                                                               
  name: csi-rbd-secret                                                  
  namespace: default                                                    
stringData:                                                             
  # Key values correspond to a user name and its key, as defined in the 
  # ceph cluster. User ID should have required access to the 'pool'     
  # specified in the storage class                                      
  userID: <plaintext ID>                                                
  userKey: <Ceph auth key corresponding to ID above>                    
                                                                        
  # Encryption passphrase                                               
  encryptionPassphrase: test_passphrase                                 
```